src/nvim/ex_docmd.c

```diff
@@ -9411,3 +9411,36 @@ static void ex_terminal(exarg_T *eap)
     xfree(name);
   }
 }
+
+static void ex_plug(exarg_T *eap)
+{
+  char *path = NULL;
+  if (*eap->arg != NUL) {
+    if (os_isdir(eap->arg)) {  // full path: plug ~/.config/nvim/plugged/coolplugin
+      path = (char *)vim_strsave_fnameescape(eap->arg, FALSE);
+    } else {  // spec: plug neovim/neovimplugin
+      char_u *user_vim = vim_strsave(p_rtp);
+      strtok((char *)user_vim, ","); // split(&runtimepath, ',')[0]
+      char_u *basename = path_tail((char_u *)eap->arg); // 'neovim/neovimplugin' -> 'neovimplugin'
```

Comment:
>  :plug puts new runtimepath to the location where it is not supposed to be.

I'm fixing this.

> VAM has absolutely no reason to call :plug because it works fine the old way.

Fair enough. Likewise, `:plug` should be able to work without there being a more featureful plugin manager. But a plugin manager should be able to use `:plug` if it wants to.